config/mysql_db.js

```diff
@@ -0,0 +1,45 @@
+// MySql
+require('dotenv').config();
+const mysql = require('mysql');
+
+const connector = mysql.createConnection({
+    host: process.env.DB_HOST,
+    password: process.env.DB_PASSWORD,
+    user: process.env.DB_USER,
+    database: process.env.DB_NAME
+});
+
+const connectToDatabase = ( ) => {
+    connector.connect(err => {
+        if(err)
+            throw err;
+        else
+            console.log('Secure');
+
+    });
+};
+
+//#region "alternative"
```

Comment:
Remove all junk lines code commented, it's not a good practice to just left code commented, and if It's needed you will need to write the explanation why is commented